### ISSUE TITLE
[Accessibility]: Code Insights Edit UI - Chart data points have no labels

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41688]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33501

### Problem description

When tabbing through data points on the chart they all announce themselves as generic links.

### Expected behavior

They should announce some sort of identifier. Like "Value 100 at Sep 1"

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @joelkw @felixfbecker @vovakulikov

**Comment by felixfbecker** [comment]
Does this really take 3 days to fix?

**Comment by vovakulikov** [comment]
@felixfbecker nope, I think 3d label is a default value for a11y issue. Since we don't use days estimation in the Code Insights team, Justin just didn't change it.
